### PR TITLE
v20.2 SQL feature support

### LINKED ISSUE #8628: Update our SQL feature support reference docs for 20.2
https://www.cockroachlabs.com/docs/stable/postgresql-compatibility.html
https://www.cockroachlabs.com/docs/stable/sql-feature-support.html

@ericharmeling, @awoods187, can you help make this update in time for the 20.2 GA?

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #8628. 

- Made some updates to SQL support pages.
- Updated some formatting/output.

I've tried to make these updates as I document a feature, but some have slipped through. Please let me know if I'm missing anything important here.

### CODE REVIEW

File: v20.2/sql-feature-support.md [hunk 1]
```diff
@@ -98,9 +98,11 @@ table tr td:nth-child(2) {
  Index renames | ✓ | Standard | [`RENAME INDEX` documentation](rename-index.html)
  Adding indexes | ✓ | Standard | [`CREATE INDEX` documentation](create-index.html)
  Removing indexes | ✓ | Standard | [`DROP INDEX` documentation](drop-index.html)
+ Altering a primary key | ✓ | Standard | [`ALTER PRIMARY KEY` documentation](alter-primary-key.html)
```
Comment (vy-ton) [inline v20.2/sql-feature-support.md:101]: Is "altering a column type" the right level for this table?

**Review by vy-ton — APPROVED** [review]

File: v20.2/sql-feature-support.md (hunk 1 above)
Comment (ericharmeling) [reply v20.2/sql-feature-support.md:102]: I believe it is. Just added it in a new row.

**Comment by ericharmeling** [comment]
@awoods187 No rush - just pinging you so you're aware of this PR.

**Comment by awoods187** [comment]
LGTM. Minor nit, can you adjust Geospatial indexes to spatial?

**Review by lnhsingh — COMMENTED** [review]
Pointed out a few lines that might need a "New in 20.2" tag but otherwise LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8725#-:-MK_PR-x9brwfnMe-6lC:bw8f8lx)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @awoods187, @ericharmeling, @lnhsingh, and @vy-ton)

___
*[v20.2/sql-feature-support.md, line 102 at r2](https://reviewable.io/reviews/cockroachdb/docs/8725#-MK_PIquA9gO1Uf2450O:-MK_PIquA9gO1Uf2450P:bhl4uvz) ([raw file](https://github.com/cockroachdb/docs/blob/f222fe5a5a692396b48cf6cb8bdd511e72fbc9e5/v20.2/sql-feature-support.md#L102)):*
> ```Markdown
>  Adding indexes | ✓ | Standard | [`CREATE INDEX` documentation](create-index.html)
>  Removing indexes | ✓ | Standard | [`DROP INDEX` documentation](drop-index.html)
>  Altering a primary key | ✓ | Standard | [`ALTER PRIMARY KEY` documentation](alter-primary-key.html)
> ```

Should this have a `New in 20.2` tag?
___
*[v20.2/sql-feature-support.md, line 173 at r2](https://reviewable.io/reviews/cockroachdb/docs/8725#-MK_PQBNE931Vhn5rUgV:-MK_PQBNE931Vhn5rUgW:bhl4uvz) ([raw file](https://github.com/cockroachdb/docs/blob/f222fe5a5a692396b48cf6cb8bdd511e72fbc9e5/v20.2/sql-feature-support.md#L173)):*
> ```Markdown
>  Information Schema | ✓ | Standard | [Information Schema documentation](information-schema.html)
>  Views | ✓ | Standard | [Views documentation](views.html)
>  Materialized views | ✓ | Common Extension | [Materialized views documentation](views.html#materialized-views)
> ```

Should this have a `New in 20.2` tag?

**Review by ericharmeling — COMMENTED** [review]
TFTR @lnhsingh !

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8725#-:-MKaEw5Y2aFfAK4Vfp_J:bhjhpnt)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @awoods187, @lnhsingh, and @vy-ton)

___
*[v20.2/sql-feature-support.md, line 102 at r2](https://reviewable.io/reviews/cockroachdb/docs/8725#-MK_PIquA9gO1Uf2450O:-MKaEMww9YmH2xmYgimb:be8o9zf) ([raw file](https://github.com/cockroachdb/docs/blob/f222fe5a5a692396b48cf6cb8bdd511e72fbc9e5/v20.2/sql-feature-support.md#L102)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Should this have a `New in 20.2` tag?
</blockquote></details>

This is actually new in 20.1, but I hadn't added it. (see `v20.1/sql-feature-support.md`)
___
*[v20.2/sql-feature-support.md, line 173 at r2](https://reviewable.io/reviews/cockroachdb/docs/8725#-MK_PQBNE931Vhn5rUgV:-MKaEWY43ijoY_pq6Wj4:b-52r88b) ([raw file](https://github.com/cockroachdb/docs/blob/f222fe5a5a692396b48cf6cb8bdd511e72fbc9e5/v20.2/sql-feature-support.md#L173)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Should this have a `New in 20.2` tag?
</blockquote></details>

Yes! Good catch.

**Comment by jseldess** [comment]
Thanks, @ericharmeling!

**Pull request merged**